x/concentrated-liquidity/tick_test.go

```diff
@@ -298,35 +298,65 @@ func (s *KeeperTestSuite) TestInitOrUpdateTick() {
 func (s *KeeperTestSuite) TestGetTickInfo() {
 	var (
 		preInitializedTickIndex = DefaultCurrTick.Int64() + 2
+		expectedUptimes         = getExpectedUptimes()
+		emptyUptimeTrackers     = wrapUptimeTrackers(expectedUptimes.emptyExpectedAccumValues)
+		varyingTokensAndDenoms  = wrapUptimeTrackers(expectedUptimes.varyingTokensMultiDenom)
 	)
 
 	tests := []struct {
-		name             string
-		poolToGet        uint64
-		tickToGet        int64
-		expectedTickInfo model.TickInfo
-		expectedErr      error
+		name                     string
+		poolToGet                uint64
+		tickToGet                int64
+		preInitUptimeAccumValues []sdk.DecCoins
+		expectedTickInfo         model.TickInfo
+		expectedErr              error
 	}{
 		{
 			name:      "Get tick info on existing pool and existing tick",
 			poolToGet: validPoolId,
 			tickToGet: preInitializedTickIndex,
-			// Note that FeeGrowthOutside is not updated.
-			expectedTickInfo: model.TickInfo{LiquidityGross: DefaultLiquidityAmt, LiquidityNet: DefaultLiquidityAmt.Neg()},
+			// Note that FeeGrowthOutside and UptimeGrowthOutside(s) are not updated.
+			expectedTickInfo: model.TickInfo{LiquidityGross: DefaultLiquidityAmt, LiquidityNet: DefaultLiquidityAmt.Neg(), UptimeTrackers: emptyUptimeTrackers},
+		},
+		{
+			name:                     "Get tick info on existing pool and existing tick with init but zero global uptime accums",
+			poolToGet:                validPoolId,
+			tickToGet:                preInitializedTickIndex,
+			preInitUptimeAccumValues: expectedUptimes.varyingTokensMultiDenom,
+			// Note that FeeGrowthOutside is not updated, but UptimeGrowthOutsides are.
+			// We expect uptime trackers to be initialized to zero since tick > active tick
+			expectedTickInfo: model.TickInfo{LiquidityGross: DefaultLiquidityAmt, LiquidityNet: DefaultLiquidityAmt.Neg(), UptimeTrackers: emptyUptimeTrackers},
```

Comment:
What do you mean by updated?

My definition of updated is when we set them to non-empty coins.

According to my interpretation, both are not updated

Reply:
I agree with your interpretation, the comment was incorrect: https://github.com/osmosis-labs/osmosis/pull/4355#discussion_r1116134261